tests/Unit/DataStructures/Test_DataVector.cpp

```diff
@@ -171,6 +171,223 @@ void check_vectors(const T1& t1, const T2& t2) {
 }
 }  // namespace
 
+SPECTRE_TEST_CASE("Unit.DataStructures.DataVector.Math",
+                  "[Unit][DataStructures]") {
+  // Test min/max/abs
```

Comment:
I disagree that this is cleaning anything up. The code was already sectioned by functions it was testing, all this does it add a lot of work for reviewers with zero benefit. If you're searching for the portion that tests `exp` you're just going to search for `exp`, you don't need additional scope and a comment to tell you where to look.